radicle-node/src/tests.rs

```diff
@@ -227,6 +228,91 @@ fn test_inventory_sync() {
     }
 }
 
+#[test]
+fn test_inventory_pruning_stale_entries() {
+    struct Test {
+        limits: Limits,
+        wait_time: LocalDuration,
+        exp_routing_table_size: usize,
```

Comment:
```suggestion
        expected_entries: usize,
```
Just a suggestion, since `exp` is not so clear, while it's clearer that we're testing the routing table.